Agent/AgentWar3/UnitRefList/Ability/AbilityInterfaced/CAbilityHero.h

```diff
@@ -16,31 +16,31 @@ class CAbilityHero : CAbilityInterfaced {
     int field_80;
     int field_84;
     int field_88;
-    int experience;
-    int skillPoints;
-    int strength; // assigned with the SetHeroStr native
-    int field_98;
-    int field_9C;
-    int field_A0;
-    int field_A4;
-    int agility; // assigned with the SetHeroAgi native
+    int heroExperience;
+    int heroLevel;
+    int heroStartingStrength;
+    float heroPrimaryAttributeValue;
+    float heroExtraHealth;
+    int heroStartingAgility;
+    float heroArmor;
+    int field_A8;
     int field_AC;
     int field_B0;
     int field_B4;
     int field_B8;
-    int intelligence; // needs to be confirmed
+    int field_BC;
     int field_C0;
     int field_C4;
-    int field_C8;
-    int field_CC;
-    int field_D0;
-    int field_D4;
-    int field_D8;
+    float heroExtraMana;
+    int heroPrimaryAttribute;   //1=str 2=int 3=agi
+    float heroStrengthPerLevel;
+    float heroIntelligencePerLevel;
+    float heroAgilityPerLevel;
     int field_DC;
     int field_E0;
     int field_E4;
     int field_E8;
-    int suspendHeroExperience;
```

Comment:
Why is this reverted back?

Reply:
Accidental, has been fixed in a new commit.

The rest of the selected fields are verified (and have been used)